samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -191,6 +193,34 @@ For example, if your development kit version is 1.0.1, use the following board n
 
 ``nrf9160dk_nrf9160_ns@1_0_1``
 
+.. _nrf_cloud_mqtt_multi_service_remote_at:
+
+Remote execution of modem AT commands
+=====================================
+
+If the :ref:`CONFIG_AT_CMD_REQUESTS <CONFIG_AT_CMD_REQUESTS>` KConfig option is left enabled, you can remotely execute modem AT commands on your device by sending a device message with appId ``MODEM``, messageType ``CMD``, and the data key set to the command you would like to execute.
+
+The application executes the command stored in the data key, and responds with a device message containing either an error code or the response given by the modem to the AT command.
+
+For example, if you send the following device message to a device running this sample with :ref:`CONFIG_AT_CMD_REQUESTS <CONFIG_AT_CMD_REQUESTS>` enabled:
+
+.. code-block:: json
+
+   {"appId":"MODEM", "messageType":"CMD", "data":"AT+CGMR"}
+
+It executes the modem AT command ``AT+CGMR`` and sends a device message similar to the following back to nRF Cloud:
+
+.. code-block:: json
+
+   {
+      "appId": "MODEM",
+      "messageType": "DATA",
+      "ts": 1669244834095,
+      "data": "mfw_nrf9160_1.3.2\r\nOK"
+   }
+
+You can do this directly from the nRF Cloud Portal by writing `{"appId":"MODEM", "messageType":"CMD", "data":"AT+CGMR"}` into the Send a message box of the Terminal card and pressing Send.
```

Comment:
```suggestion
In the nRF Cloud portal, write `{"appId":"MODEM", "messageType":"CMD", "data":"AT+CGMR"}` into the **Send a message** field of the **Terminal** card and click :guilabel:`Send`.
```

Reply:
I do want to make sure to acknowledge that there are other ways beside the cloud portal, so I have deviated slightly, but otherwise tried to apply your changes